spec/measure_repository_service_test_kit/read_by_id_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+RSpec.describe MeasureRepositoryServiceTestKit::ReadById do
+  let(:suite) { Inferno::Repositories::TestSuites.new.find('measure_repository_service_test_suite') }
+  let(:group) { suite.groups[1] }
+  let(:session_data_repo) { Inferno::Repositories::SessionData.new }
+  let(:test_session) { repo_create(:test_session, test_suite_id: suite.id) }
+  let(:url) { 'http://example.com/fhir' }
+  let(:error_outcome) { FHIR::OperationOutcome.new(issue: [{ severity: 'error' }]) }
+
+  def run(runnable, inputs = {})
+    test_run_params = { test_session_id: test_session.id }.merge(runnable.reference_hash)
+    test_run = Inferno::Repositories::TestRuns.new.create(test_run_params)
+    inputs.each do |name, value|
+      session_data_repo.save(test_session_id: test_session.id, name:, value:, type: 'text')
+    end
+    Inferno::TestRunner.new(test_session:, test_run:).run(runnable)
+  end
+
+  describe 'Server successfully retrieves specified resource by its id' do
+    let(:test) { group.tests.first }
+    let(:resource_id) { 'measure_id' }
+    let(:resource_type) { 'Measure' }
```

Comment:
It seems like the rspec tests only check if the resource type is Measure, would it be worth it or redundant to add the same for Library resources?